challenges/objects-arrays.js

```diff
@@ -46,26 +56,48 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 {"id":9,"first_name":"Michail","university":"Universidad Católica de Ávila","email":"mrome8@shinystat.com"},
 {"id":10,"first_name":"Hube","university":"Universitat Rovira I Virgili Tarragona","email":"hlethbrig9@foxnews.com"}]
 
-/* Request 1: Create a new array called universities that contains all the universities in the graduates array.  
+//Request 1: Create a new array called universities that contains all the universities in the graduates array.
+
+let universities = graduates.map((unis) => {
+  return unis.university;
+});
+console.log(universities); 
 
-Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
-console.log(universities)
 
-/* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
+// Once you have the new array created, sort the universities alphabetically and log the result. */
 
-The resulting contact information should have a space between the first name and the email information like this: 
-Name email@example.com
+universities.sort();
 
-Log the result of your new array. */
-const contactInfo = [];
+console.log(universities);
+
+
+//Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
+
+let contactInfo = [];
+let emailName = graduates.forEach(function(i){
+  contactInfo.push(i.first_name + " " + i.email);
+});
 console.log(contactInfo);
 
+// The resulting contact information should have a space between the first name and the email information like this: 
+// Name email@example.com
+
+// Log the result of your new array. */
+// const contactInfo = [];
+// console.log(contactInfo);
+
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
-const uni = [];
-console.log(uni);
+// 
+
 
+// let uni = graduates.filter(function(unionly) {
+//   return unionly == graduates.university;
+// });
+
+const allUnis = graduates.map(x => x.university.match(/Uni/g));
+
+console.log(allUnis);
 
```

Comment:
Your console.log is only returning nul and 'uni' instead of the names of the universities that contain uni, see if you can fix this